src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="searchbar" data-qa="big">
+      <img
+        class="magnifier_big"
+        src="images/Search.svg"
+        alt="lupa"
+      >
+      <input
+        class="big_bar bars"
```

Comment:
this is a single bar, not bars. Also, etter to create more descriptive names like `search-bar` `search-input` etc.